src/plugins/workspace/public/components/workspace_creator/workspace_icon_selector.tsx

```diff
@@ -18,19 +16,30 @@ export const WorkspaceIconSelector = ({
   value?: string;
   onChange: (value: string) => void;
 }) => {
-  return (
-    <EuiFlexGroup>
-      {icons.map((item) => (
-        <EuiFlexItem
-          key={item}
-          onClick={() => {
-            onChange(item);
-          }}
-          grow={false}
-        >
-          <EuiIcon size="l" type={item} color={value === item ? color : undefined} />
+  const icons = ['Glasses', 'Search', 'Bell', 'Package'];
```

Comment:
I'd recommend moving this statement out of the function body if it is a static property.